packages/package-requester/src/packageRequester.ts

```diff
@@ -480,7 +481,7 @@ function fetchToStore (
             }
           }),
       )
-      await writeJsonFile(pkgIndexFilePath, integrity)
+      await writeJsonFile(pkgIndexFilePath, { files: integrity }, { indent: undefined })
```

Comment:
if `indent` is not set, the default indent is tabs. We don't need indentation in these files.